src/component/MediaCard/index.js

```diff
@@ -1 +1,40 @@
-// 여기다가 만드시면 됩니다!!!
+import React from "react";
+import StyledImage from "../Image";
+import CardWrapper from "../CardWrapper";
+import TextBox from "../Text";
+import StyledButton from "../Button";
+import StyledCardWrapper from "../CardWrapper";
+
+const MediaCard = (props) => {
+  return (
+    <StyledCardWrapper border_radius="20px" background_color="transparent">
+      <StyledImage
+        border_radius="20px"
+        backgroundimage={props.backgroundimage}
+      ></StyledImage>
+      <TextBox
```

Comment:
```suggestion
    <StyledCardWrapper border_radius="20px" background_color="transparent">
      <StyledImage
        border_radius="20px"
        backgroundimage={props.backgroundimage}
      />
      <TextBox
```

여는태그와 닫는태그 사이에 아무 내용도 없다면 self-closing tag 를 사용해볼 수 있을 것 같아요. 위와 같이 수정하면 더 좋은 코드가 될 것 같아요 😎